runtime/vm/callin.cpp

```diff
@@ -1341,6 +1344,12 @@ jitFillOSRBuffer(struct J9VMThread *currentThread, void *osrBlock)
 void JNICALL
 initializeAttachedThread(J9VMThread *currentThread, const char *name, j9object_t *group, UDATA daemon, J9VMThread *initializee)
 {
+#if defined(J9VM_OPT_CRIU_SUPPORT)
```

Comment:
Im wondering if I should just acquire the `vmThreadListMutex` to make this case impossibl